app.js

```diff
@@ -39,19 +39,31 @@ var app = new Vue({
                     totalHiatus: 0
                 }
             },
+            show_defrag: false,
+            defrag: false,
             show_arcs: false,
             info: {},
             major_hiatuses: []
         },
         computed: {
             issues_by_year: function () {
+                var defrag = this.defrag
                 var data = [];
                 var t = _.groupBy(this.all_issues, 'year');
                 Object.keys(t)
                     .sort()
                     .reverse()
                     .forEach(function (key) {
-                        data.push({year: key, issues: _.sortBy(t[key], ['number'])})
+                        data.push({
+                          year: key,
+                          issues: _.sortBy(t[key], (issue) => {
+                            if (defrag) {
+                              return issue.number + (issue.released ? 0 : 60);
+                            } else {
+                              return issue.number;
+                            }
+                          }),
+                        });
```

Comment:
Instead of a magic number you can replace the sort with:

```javascript
data.push({year: key, issues: _.orderBy(t[key], ['released', 'number'], ['desc', 'asc'])})
```

I think it makes it more clear what's happening.

Reply:
So I still have to check for defrag, which syntax are you most comfortable with?
```js
const issues = defrag ? _.orderBy(t[key], ['released', 'number'], ['desc', 'asc']) : _.orderBy(t[key], ['number'], ['asc']) 
data.push({ year: key, issues })
```
or 
```js
if (defrag) {
    data.push({ year: key, issues: _.orderBy(t[key], ["released", "number"], ["desc", "asc"]) });
} else {
    data.push({ year: key, issues: _.orderBy(t[key], ["number"], ["asc"]) });
}
```

Reply:
The later for sure.